integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -239,10 +231,9 @@ func (rcs *KeeperRegistrySettings) EncodeOnChainConfig(registrar string) ([]byte
 			Registrars:             []common.Address{common.HexToAddress(registrar)},
 			UpkeepPrivilegeManager: common.Address{},
 		}
-		encodedOnchainConfig, err := utilsABI.Pack("_onChainConfig", &onchainConfigStruct)
+		encodedOnchainConfig, err := utilsABI.Methods["_onChainConfig"].Inputs.Pack(&onchainConfigStruct)
```

Comment:
nice :sunglasses:

Reply:
Is `.Methods` generally considered safer than `.Pack`?

Reply:
When we just use the Pack function, we have to slice off the first 4 bytes which is the function signature. We dont like the magic number 4 and it is easy to forget too.
Using this Methods function, we dont need to remember the magic 4 bytes any more :)